### PR TITLE
UAF-3334 Active / Inactive fields for Project Directors are not displ…

### PR DESCRIPTION
**Pull request by mwellman11**
…ayed on Proposal Inquiry Screen

### CODE REVIEW

**Comment by ghost** [comment]
This change should have been made to an edu.arizona version of the file. This isn't one of those case where we need to modify the delivered code. See Award.xml BO DD file in edu.arizona for how this could be done.

**Comment by mwellman11** [comment]
I'll get that changed

**Comment by mwellman11** [comment]
Requested changes have been made

File: kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Proposal.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+    
+    <bean
```
Comment (ghost) [inline kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Proposal.xml]: Should be:
`<bean id="Proposal-inquiryCollectionDefinition-proposalProjectDirectors"
		parent="Proposal-inquiryCollectionDefinition-proposalProjectDirectors-parentBean">`
We override the child bean, not the parent bean.

**Review by ghost — COMMENTED** [review]
Second review complete. Please see my comment.

File: kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Proposal.xml (hunk 1 above)
Comment (mwellman11) [reply kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Proposal.xml]: Requested Change has been made

**Comment by ghost** [comment]
Looks good!

**Pull request merged**